kfs-core/src/main/java/edu/arizona/kfs/fp/document/web/struts/GeneralErrorCorrectionAction.java

```diff
@@ -401,19 +409,13 @@ public ActionForward insertTargetLine(ActionMapping mapping, ActionForm form, Ht
     }
 
 
-    /*
-     * This is the "import" button in the "reversing" section, so we know this will always be
-     * a target line -- the "reversing" section does not have import, since the GLE always
-     * comes off of a GLE search. Still popped it into an if/else just in case someone else down
-     * the line uses this.
-     *
-     * Note: this also does the reversing of the flags
-     */
     @Override
     protected void insertAccountingLine(boolean isSource, KualiAccountingDocumentFormBase financialDocumentForm, AccountingLine line) {
 
-        // Only set d/c flag for target import, just in case this gets called in future code w/ a source line
-        if (!isSource) {
+        // 1. !isSource: Source lines will have their d/c code set from its flipped GLE, leave it
+        // 2. line.getDebitCreditCode() == null: This line is imported from file, so needs a d/c code since the import template
+        //                                       doesn't have one; we'll attempt to infer from the source lines

```

Comment:
I'd add an additional comment here: if all the source lines have THE SAME DebitCreditCode then it can be inferred to that particular one, otherwise exception will be thrown.